docs/source/partial-transformers/partial-transformers.rst

```diff
@@ -166,6 +166,54 @@ You can expect that basic functionality of chimney's ``Transformer`` either work
 fashion in ``PartialTransformer``\s, or have some counterparty methods in the API
 (usually with the `Partial` prefix or suffix in the name).
 
+Preferring total or partial transformer
```

Comment:
Haven't we considered moving that to a separate page?

Reply:
done